source/particles/mus_particle_config.fpp

```diff
@@ -0,0 +1,1892 @@
+!                                       & nChunkVals     = nChunkVals,     &
+!                                       & key            = 'velocity',     &
+!                                       & iStart         = iStart          )
+!      
+!      call mus_load_particle_data_chunk( conf           = conf,           & 
+!                                       & parent         = parent,      &
+!                                       & particleBuffer = particleBuffer_MEM, &
+!                                       & nChunkVals     = nChunkVals,     &
+!                                       & key            = 'force',        &
+!                                       & iStart         = iStart          )
+!
+!      call mus_load_particle_data_chunk( conf           = conf,           & 
+!                                       & parent         = parent,      &
+!                                       & particleBuffer = particleBuffer_MEM, &
+!                                       & nChunkVals     = nChunkVals,     &
+!                                       & key            = 'radius',       &
+!                                       & iStart         = iStart          )
+!
+!      call mus_load_particle_data_chunk( conf           = conf,           & 
+!                                       & parent         = parent,      &
+!                                       & particleBuffer = particleBuffer_MEM, &
+!                                       & nChunkVals     = nChunkVals,     &
+!                                       & key            = 'mass',         &
+!                                       & iStart         = iStart          )
+!
+!
+!      ! nChunkVals is the number of particles actually read from the lua file
+!      ! Can be less than chunkSize if there are fewer than chunkSize particles
+!      ! left in lua table
+!      ! write(stdOutUnit,*) 'nChunkVals = ', nChunkVals
+!    
+!      !-- SET PARTICLE ROTATIONAL INERTIA --!
+!      do iParticle = 1, nChunkVals
+!        particleBuffer_MEM( iParticle )%rotInertia & 
+!          & = 0.4 * particleBuffer_MEM( iParticle )%mass &
+!          &       * particleBuffer_MEM( iParticle )%radius**2 
+!
+!      end do
+!
+!      ! Append particles in particleBuffer to particleGroup
+!      ! write(stdOutUnit,*) 'Loading particles: nChunkVals = ', nChunkVals
+!      do iParticle = 1, nChunkVals
+!        wasAdded = .FALSE.
+!        ! write(stdOutUnit,*) 'Appending particle'
+!
+!        call append_DA_particle_MEM( me        = particles,                       &
+!                                   & particle  = particleBuffer_MEM(iParticle),   &
+!                                   & length    = 1,                               &
+!                                   & wasAdded  = wasAdded                         ) 
+!
+!      end do
+!
+!      iStart = iStart + chunkSize
+!    end do chunkLoop
+!
+!    deallocate( particleBuffer_MEM )
+!
+!  end subroutine load_particle_data_MEM
```

Comment:
Do we need this commented out routine? I'd prefer to not include commented out code.